test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -93,6 +93,63 @@ static public void testCreatePoint2D() throws Throwable {
 		assertEquals(getY.invoke(point2D), yNew);
 	}
 
+	/*
+	 * Test array of Point2D
+	 */
+	@Test(priority=1)
```

Comment:
if you make this priority 2 then `point2DClass`, `makePoint2D` and the associated getters/setters will already exist